### PR TITLE
force eslint error

### PR DESCRIPTION
**Pull request by Amwam**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,3 +1,7 @@
 {
-    "extends": "react-app"
+    "extends": "react-app",
```
Comment (markstory) [inline .eslintrc:2]: Mark from Stickler-CI here. Currently we don't support the `react-app` eslint preset. I can look into getting support for that preset added though.

Comment (Amwam) [reply .eslintrc:2]: Ah, that makes sense. Is there a way to get more output from stickler?
What presets do you support (if any)? I was unsure if I was getting the .stickler.yml wrong, or something else

Comment (markstory) [reply .eslintrc:2]: Right now the only presets that are supported are those built directly into eslint, and those that come with your application code. I'm going to make the error messages for eslint include missing presets in the build status soon.

Other than 'react-app' are there other presets that react developers often use?

Comment (Amwam) [reply .eslintrc:2]: Thanks!

I'm not sure about presets for react. There is often the 'extends' of other configs, just as `eslint-config-airbnb` I've not tried to see if that works.

Comment (markstory) [reply .eslintrc:2]: Cool. I'll look into adding those presets.

**Pull request closed without merging**